projects/components/src/datagrid/datagrid.component.scss

```diff
@@ -19,6 +19,28 @@ $supported-buttons: (0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 1
     }
 }
 
+:host {
+    &.fill-parent {
+        flex: 1;
+        display: flex;
+        flex-direction: column;
+    }
+}
+
+.fill-parent {
+    flex: 1;
+    display: flex;
+    flex-direction: column;
+}
+
+.set-height {
+    height: var(--datagrid-height) !important;
```

Comment:
Please add a comment explaining that this is to work around a Clarity bug. It could be this one. https://github.com/vmware/clarity/issues/2364 If it's not (since this doesn't talk about number of rows and page size being the same) and we can repro the problem on Clarity V3, we should file a bug

Reply:
I tried for ~2 hours, and I couldn't reproduce this bug in a simpler environment and I couldn't figure it out. I added a comment talking about a bug, but because I can't reproduce it in a Stackblitz workspace, it can't submit a GitHub issue.